### ISSUE TITLE
Used custom block issue

### ISSUE THREAD

**Issue by subhash16** [Sketchware-Pro/Sketchware-Pro#1495]
See previously its working like this
![Screenshot_20241210-211510~2](https://github.com/user-attachments/assets/2fb12d64-6749-4b27-b6c4-02239b6a59ba)
Now there is no option that showing Missing block or like that
![Screenshot_20241210-211624](https://github.com/user-attachments/assets/8480fb77-02bd-41fe-a230-c7c7097ca080)

**Comment by subhash16** [comment]
@Edward-NewGate-KOC

**Comment by subhash16** [comment]
Also count of used custom block not showing

**Comment by Edward-NewGate-KOC** [comment]
1. Soon, I will add a count for used custom blocks just like the old versions

2. ( Found in data/.......) it's unnecessary

3. If the block is missing, it's already showing that for you. Just try it

**Comment by Edward-NewGate-KOC** [comment]
Is there anything else you'd like to say?

**Comment by subhash16** [comment]
No

**Comment by trindadedev13** [comment]
Is this really a issue?

**Comment by elfilibusterismo** [comment]
> 1. Soon, I will add a count for used custom blocks just like the old versions
> 2. ( Found in data/.......) it's unnecessary
> 3. If the block is missing, it's already showing that for you. Just try it

Agree, but having to know which is missing is better as they will know what to import, and some of the additional built-in blocks(created by sketchware pro) is showing there too. And that shouldn't be happen.

Anyways, It might be better to add the activities where the block are located.

**Comment by subhash16** [comment]
@Edward-NewGate-KOC Please Sir It's a hardly request that please add a dialog when opening a project that your Custom Block Missing if the block is missing.
And also the missing block list.

**Comment by Edward-NewGate-KOC** [comment]
> [@Edward-NewGate-KOC](https://github.com/Edward-NewGate-KOC) Please Sir It's a hardly request that please add a dialog when opening a project that your Custom Block Missing if the block is missing. And also the missing block list.

If we add it your way, the page will load slower,
so I just put it inside the same dialog — works better like that

**Comment by subhash16** [comment]
@Edward-NewGate-KOC need to show a dialog that custom block missing because check box is so small that some time not visible

**Comment by subhash16** [comment]
![Image](https://github.com/user-attachments/assets/499125ad-85ef-4abf-8749-e27e7a80b8f4)
1)✓✓✓ """Need missing block count also"""
When backup (include used Custom Block tick mark) and if i remove custom block and restore the project its showing like this ,Need to show Missing.Like you have implemented if block removed without backup its showing Missing.

**Comment by subhash16** [comment]
![Image](https://github.com/user-attachments/assets/478416a7-20b9-42ee-ab7c-1317c7ee3392)
I think I didn't tell you correctly.

See this picture 
1) First you create a custom block 
2) then implement it on any projects
3)  now backup like above the picture and delete the project 
4) now delete the custom block and restore the project 
Then see what happened.
In this situation there is no toast showing that missing custom block or some kind of toast like found in backup but not found in block list
@Edward-NewGate-KOC

**Comment by Edward-NewGate-KOC** [comment]
> See this picture
> 
> 1. First you create a custom block
> 2. then implement it on any projects
> 3. now backup like above the picture and delete the project
> 4. now delete the custom block and restore the project
>    Then see what happened.
>    In this situation there is no toast showing that missing custom block or some kind of toast like found in backup but not found in block list
>    [@Edward-NewGate-KOC](https://github.com/Edward-NewGate-KOC)

That's what I'm talking about, i think you did something wrong, can record a video?
